packages/generator-react-server/generators/app/templates/test.js

```diff
@@ -0,0 +1,54 @@
+import cp from 'child_process';
+import http from 'http';
+import test from 'ava';
+
+test.before('start the server', async () => {
+	let stdout = await exec('npm run compile');
+	console.log(stdout);
+	stdout = await exec('node node_modules/.bin/pm2 start npm -i 1 --name "hello-world" -- start');
```

Comment:
> bring the server down

[Kill](https://nodejs.org/api/child_process.html#child_process_child_kill_signal) it!

> tail the logs in real time in a separate window

Is that a requirement for automated tests?